en/mozorg/about/manifesto.ftl

```diff
@@ -0,0 +1,123 @@
+manifesto-use-these-free-tools-to = Use these free tools to teach the web
+manifesto-learn-about-creating-and = Learn about creating and curating content for the web
+manifesto-add-new-voices-to-open = Add new voices to open source technology
+manifesto-set-your-do-not-track = Set your Do Not Track preference
+manifesto-understand-the-web-ecosystem = Understand the web ecosystem
+manifesto-explore-how-open-practices = Explore how open practices keep the web accessible
+manifesto-learn-how-to-remix-content = Learn how to remix content to create something new
+manifesto-learn-how-to-maximize = Learn how to maximize the interactive potential of the web
+manifesto-participate-in-our-governance = Participate in our governance forum
+manifesto-join-us-as-a-volunteer = Join us as a volunteer
+manifesto-learn-how-to-collaborate = Learn how to collaborate online
+manifesto-visualize-who-you-interact = Visualize who you interact with on the web with Lightbeam
+manifesto-learn-about-creating-web = Learn about creating web resources with others
+manifesto-host-or-join-a-maker-party = Host or join a Maker Party
+manifesto-learn-how-to-build-online = Learn how to build online collaboration skills
+manifesto-read-the-entire-manifesto = Read the entire manifesto
+manifesto-love-the-web = Love the web?
+manifesto-get-the-mozilla-newsletter = Get the { -brand-name-mozilla } newsletter and help us keep it open and free.
+manifesto-details-the-mozilla-manifesto = The { -brand-name-mozilla } Manifesto
+manifesto-details-introduction = Introduction
+manifesto-details-the-internet-is-becoming = The internet is becoming an increasingly important part of our lives.
+manifesto-details-the-mozilla-project-global = The { -brand-name-mozilla } project is a global community of people who believe that openness, innovation, and opportunity are key to the continued health of the internet. We have worked together since 1998 to ensure that the internet is developed in a way that benefits everyone. We are best known for creating the { -brand-name-mozilla } { -brand-name-firefox } web browser.
+manifesto-details-the-mozilla-project-community = The { -brand-name-mozilla } project uses a community-based approach to create world-class open source software and to develop new types of collaborative activities. We create communities of people involved in making the internet experience better for all of us.
+manifesto-details-as-a-result-of = As a result of these efforts, we have distilled a set of principles that we believe are critical for the internet to continue to benefit the public good as well as commercial aspects of life. We set out these principles below.
+manifesto-details-the-goals-for = The goals for the Manifesto are to:
+manifesto-details-articulate-a-vision = articulate a vision for the internet that { -brand-name-mozilla } participants want the { -brand-name-mozilla } Foundation to pursue;
```

Comment:
`the { -brand-name-mozilla } Foundation` -> `the { -brand-name-mozilla-foundation }`